docs/content_management/work_with_forms.md

```diff
@@ -65,6 +69,10 @@ You can choose one of a few options of what is shown to the user after filling i
 
     For more information, see [Publish content](publish_content.md).
 
+**Check out the interactive demo below:**
```

Comment:
You've used this phrase in one other place, and not in the others, so why not skip it altogether?